Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -212,13 +261,27 @@ - (NSDictionary *)constantsToExport
   [RCTSharedApplication() presentLocalNotificationNow:notification];
 }
 
-RCT_EXPORT_METHOD(scheduleLocalNotification:(UILocalNotification *)notification)
+RCT_EXPORT_METHOD(scheduleLocalNotification:(UILocalNotification *)notification callback:(RCTResponseSenderBlock)callback)
 {
   [RCTSharedApplication() scheduleLocalNotification:notification];
+
+  NSString *identifier = [[NSUUID UUID] UUIDString];
+  [_scheduledLocalNotifications setValue:notification forKey:identifier];
+  callback(@[identifier]);
```

Comment:
I think it's more common to use a monotonically increasing numeric value for identifiers like this in JS-world. Since these ids only seem to last for the lifetime of the app running, I don't see any reason why they need to be UUIDs, so an NSNumber would be cheaper.